srv/GetGroupUrdf.srv

```diff
@@ -0,0 +1,4 @@
+string group_name # Name of the planning group for which an URDF is requested
+---
+bool success   # indicate successful run of the service
```

Comment:
Should we add an error_message field, to be filled in if the request fails?

Reply:
Good point! Currently, I publish the error details via the log in the move_group node but we can change that to a [MoveItErrorMsg](https://github.com/ros-planning/moveit_msgs/blob/ros2/msg/MoveItErrorCodes.msg) and add more information :+1: